base-template.md

```diff
@@ -309,9 +312,9 @@ Tables 1 and 2 also document conformance with:
 <td><!-- INSERT CONTENT HERE --></td>
 </tr>
 <tr id="character-key-shortcuts" valign="top">
-<td><a href="https://www.w3.org/TR/WCAG21/#character-key-shortcuts"><strong>2.1.4 Character Key Shortcuts</strong></a> (Level A 2.1 only)
+<td><a href="https://www.w3.org/TR/WCAG21/#character-key-shortcuts"><strong>2.1.4 Character Key Shortcuts</strong></a> (Level A 2.1 and 2.2)
```

Comment:
My initial thought was, "Should this now link here?" (`WCAG22` not `WCAG21`)
https://www.w3.org/TR/WCAG22/#character-key-shortcuts

But now I have seen more of the document, it tends to link to the earlier versions of the guidelines if they apply to more than one version, so I guess this is fine.